packages/api/src/routes/endpoints/control/controlReject.js

```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright 2020-2024 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+async function controlReturn(context, { control }) {
```

Comment:
This should be `controlReject`

Reply:
Thanks for catching that. Resolved in 9f6ce315b1fb70a019c468f4f309aec4260f909f